packages/loot-core/src/server/budget/app.js

```diff
@@ -12,15 +12,10 @@ app.method(
 );
 app.method('budget/set-zero', mutator(undoable(actions.setZero)));
 app.method('budget/set-3month-avg', mutator(undoable(actions.set3MonthAvg)));
-app.method('budget/set-all-future', mutator(undoable(actions.setAllFuture)));
```

Comment:
This is still being used. You should either revert this removal or clean it up entirely.

Reply:
I'm looking.  I'm not particularly efficient at JS, so I'll be little slow.  Thanks for your patience.

Reply:
No problem. Take all the time you need.

What I recommend is doing a global search for the function names you're removing and see what comes up. You can either do that in your IDE or in Github.

Here's an example search in Github: https://github.com/actualbudget/actual/search?q=set-all-future

By following this approach you'll find all the references to the functions and hence will be able to remove all of them.

Reply:
I saw this comment: "setAllFuture only applies to report budget type"
I'm not familiar with that Budget Type.  Are we sure this should be removed?